fdpa-resolutions.html

```diff
@@ -86,5 +28,159 @@ <h2>Fair Districts PA - Local Resolutions</h2>
+    </script>
+    <script type="text/javascript">
+      var map;
+      var layer_0;
+      var layer_1;
+      function initialize() {
+        map = new google.maps.Map(document.getElementById('map-canvas'), {
+          center: new google.maps.LatLng(40.99058069667954, -77.62118299218751),
+          zoom: 8,
+          zoomControl: true,
+              zoomControlOptions: {
+              position: google.maps.ControlPosition.TOP_LEFT
+          },
+          fullscreenControl: true,
+          streetViewControl: false,
+          mapTypeControl: false
+        });
+        var style = [
+          {
+            featureType: 'all',
+            elementType: 'all',
+            stylers: [
+              { saturation: -85 }
+            ]
+          },
+          {
+            featureType: 'poi',
+            elementType: 'all',
+            stylers: [
+              { visibility: 'off' }
+            ]
+          }
+        ];
+        var styledMapType = new google.maps.StyledMapType(style, {
+          map: map,
+          name: 'Styled Map'
+        });
+        map.mapTypes.set('map-style', styledMapType);
+        map.setMapTypeId('map-style');
+        layer_0 = new google.maps.FusionTablesLayer({
+          query: {
+            select: "col12>>0",
+            from: "1emKZpIHeBPHfcWjdckvWJ_DWmgNrJYTRAMrmllzJ",
+            where: " col3>>0 = 'County'"
+          },
+          map: map,
+          styleId: 3,
+          templateId: 4
+        });
+        layer_1 = new google.maps.FusionTablesLayer({
+          query: {
+            select: "col16>>0",
+            from: "1emKZpIHeBPHfcWjdckvWJ_DWmgNrJYTRAMrmllzJ",
+            where: " col3>>0 in ('Borough', 'City', 'Town', 'Township (1st)', 'Township (2nd)', 'Organization') and col5>>0 in ('In work', 'Materials sent', 'Passed')"
+          },
+          map: map,
+          styleId: 4,
+          templateId: 5
+        });
+      }
+      (function() {
+        var apiKey = {
+          amyprime: 'AIzaSyBxdPyKNXkcFbFPs5I_e-xqGCUPGVuyabs',
+          devel: 'AIzaSyBz3CQw1aK0S9UHwxKq-YZtIeqdcOby-Jc'
+        };
+        var locationUrl = new URL(window.location.href);
+
+        addGoogleMaps();
+
+        var statuses = ['passed', 'inWork'];
+        var countElIds = {
+          fallbackCount: 'fall-back-count',
+          currentCount: 'current-count'
+        };
+        var windowPromise =
+          !!document.getElementById(countElIds.fallbackCount) ?
+          Promise.resolve(true) :
+          new Promise(function(windowLoaded) {
+            window.addEventListener('load', windowLoaded);
+          });
+
+        new Promise(function(scriptAdded){
+          addScript(isProduction() ?
+            'https://cdn.rawgit.com/RobinIsTheBird/fdpa-fusiontables/master/distribution/fusionApi_es5.js' :
+            'https://rawgit.com/RobinIsTheBird/fdpa-fusiontables/master/distribution/fusionApi_es5.js',

```

Comment:
I'm not seeing any difference between the two fusionApi_es5.js files (though I didn't fully diff them).  Why the switch?  This may be worth adding a comment.

Reply:
Sounds like I need to add a comment here. If you visit https://rawgit.com/, you'll see that it converts a github URL into not one but two rawgit URLs, one to use in production, the other to use in development. That's what this switch is all about. They both fetch the exact same .js file from github.

Reply:
Ah, yup, that site explains it nicely.  A quick comment pointing people there would be helpful.

Reply:
How about something like:
```
/*
 * See rawgit.com for an explanation of why two URLs are necessary here.
 */